documentation/tutorial/07-deploy-ingress.md

```diff
@@ -0,0 +1,183 @@
+temporal-k8s/0*              active    idle   10.1.232.64
+temporal-ui-k8s/0*           active    idle   10.1.232.72
+
+```
+
+## Verify Ingress Resource
+
+To verify the ingress resources were correctly created, you can run the
+following command:
+
+```bash
+kubectl describe ingress -n temporal-model
+```
+
+The output should look similar to the following (with the exception of the
+service IP addresses):
+
+```
+Name:             temporal-k8s-ingress
+Labels:           app.juju.is/created-by=nginx-ingress-integrator
+                  nginx-ingress-integrator.charm.juju.is/managed-by=nginx-ingress-integrator
+Namespace:        temporal-model
+Address:          127.0.0.1
+Ingress Class:    public
+Default backend:  <default>
+TLS:
+  temporal-tls terminates temporal-k8s
+Rules:
+  Host          Path  Backends
+  ----          ----  --------
+  temporal-k8s
+                /   temporal-k8s-service:7233 (10.1.232.64:7233)
+Annotations:    nginx.ingress.kubernetes.io/backend-protocol: GRPC
+                nginx.ingress.kubernetes.io/proxy-body-size: 20m
+                nginx.ingress.kubernetes.io/proxy-read-timeout: 60
+                nginx.ingress.kubernetes.io/rewrite-target: /
+Events:         <none>
+
+
+Name:             temporal-ui-k8s-ingress
+Labels:           app.juju.is/created-by=nginx-ingress-integrator
+                  nginx-ingress-integrator.charm.juju.is/managed-by=nginx-ingress-integrator
+Namespace:        temporal-model
+Address:          127.0.0.1
+Ingress Class:    public
+Default backend:  <default>
+TLS:
+  temporal-tls terminates temporal-ui-k8s
+Rules:
+  Host             Path  Backends
+  ----             ----  --------
+  temporal-ui-k8s
+                   /   temporal-ui-k8s-service:8080 (10.1.232.72:8080)
+Annotations:       nginx.ingress.kubernetes.io/backend-protocol: HTTP
+                   nginx.ingress.kubernetes.io/proxy-body-size: 20m
+                   nginx.ingress.kubernetes.io/proxy-read-timeout: 60
+                   nginx.ingress.kubernetes.io/rewrite-target: /
+Events:            <none>
+```
+
+## Connect Ingress
+
+Once deployed and related, find the IP of the ingress controller by running the
+following command:
+
+```bash
+kubectl get pods -n ingress -o wide
+```
+
+You should see the following output:
+
+```
+NAME                                      READY   STATUS    RESTARTS          AGE    IP           NODE      NOMINATED NODE   READINESS GATES
+nginx-ingress-microk8s-controller-mfmtx   1/1     Running   512 (3h15m ago)   145d   10.1.232.8   ubuntu   <none>           <none>
+```
+
+and add the IP-to-hostname mapping in your /etc/hosts file as follows:
+
+```bash
+sudo nano /etc/hosts
+
+10.1.232.8     temporal-k8s
+10.1.232.8     temporal-ui-k8s
+```
+
+By default, the hostname will be set to the respective application names
+`temporal-k8s` and `temporal-ui-k8s`. You can then connect a Temporal client
+through this hostname i.e. `Client.connect("temporal-k8s")`.
```

Comment:
You could also add the address of the UI now

Reply:
Actually, when I try an access the UI now via the ingress I get `SSL certificate "temporal-model/temporal-tls" does not contain a Common Name or Subject Alternative Name for server "temporal-ui-k8s": x509: certificate is valid for temporal-k8s, not temporal-ui-k8s`.

Reply:
This will be resolved with a PR i'll be creating for the UI charm soon.